.travis.yml

```diff
@@ -21,11 +21,12 @@ addons:
     - protobuf-compiler
     - python-protobuf
 before_install:
-  - export CHROME_BIN=google-chrome
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-  - sudo dpkg -i google-chrome*.deb
+  - mkdir chrome
+  - dpkg -x google-chrome*.deb chrome
```

Comment:
I just tried these steps, and the filename that gets saved is `google-chrome-stable_current_amd64.deb`. I don't see why we shouldn't use the full filename here.

`dpkg -x google-chrome-stable_current_amd64.deb $TRAVIS_BUILD_DIR/chrome`

/cc @erwinmombay, in case there was a reason to use a wildcard for the chrome .deb